README.md

```diff
@@ -101,6 +109,18 @@ import Account from "tasit-account";
 
 ...with the rest of the code remaining the same.
 
+Or maybe you just want to use the simple abstraction for interacting with smart contracts. Then just install `tasit-action` directly instead.
+
+```
+npm install --save tasit-action
+```
+
+```javascript
+import Contract from "tasit-action";
```

Comment:
Is that:
`import { Contract } from "tasit-action"`
the correct way?

Reply:
It crossed my mind while typing it that I might have it wrong. Thanks for pointing this out. Also feel free to use the GitHub suggestions feature so I can insta-accept it

![suggestion](https://user-images.githubusercontent.com/2731712/51626949-56eabd00-1f06-11e9-8235-9c69a9fbc714.jpg)

Reply:
Thanks.